onnx/external_data_helper.py

```diff
@@ -78,6 +78,8 @@ def set_external_data(tensor,  # type: TensorProto
                       checksum=None,  # type: Optional[Text]
                       basepath=None  # type: Optional[Text]
                       ):  # type: (...) -> None
+    if not tensor.HasField("raw_data"):
```

Comment:
Unclear if this change is required here. It seems to be a minor utility; may be better to not even expose this externally. But if it is used externally, the correctness depends on the usage, I suppose. But I am okay with adding this as extra check for now.